File: patern1.java
import java.util.Scanner;
public class patern1{
    public static void main(String[] args){
       Scanner akash = new Scanner(System.in);
       System.out.print("Enter the value of row : ");
        int row = akash.nextInt();
       System.out.print("Enter the number of column : ");
        int col = akash.nextInt();
       for(int i=1; i<=row; i++){
        for(int j=1; j<=col; j++){
            if(i==1 || i==row || j==1 || j==col){
                System.out.print("*");
            }
            else{
                System.out.print(" ");
            }
        }
        System.out.print("\n");
       }
    }
}

File: patern1.c
#include<stdio.h>

int main(){
    int row ,col;
    printf("Enter the number of row : ");
    scanf("%d",&row);
    printf("Enter the number of column : ");
    scanf("%d",&col);
    for(int i=1;i<=row; i++){
        for(int j=1; j<=col; j++){
            if(i==1 || i==row){
                printf("*");
            }
            else if (j==1 || j==col)
            {
                printf("*");
            }
            else{
                printf(" ");
            }
            
        }
        //printf("*");
        printf("\n");
    }
return 0;
}